collect_app/src/main/java/org/odk/collect/android/activities/FormDownloadList.java

```diff
@@ -286,7 +276,7 @@ private void clearChoices() {
     @Override
     protected void onListItemClick(ListView l, View v, int position, long id) {
         super.onListItemClick(l, v, position, id);
-        mDownloadButton.setEnabled(!(selectedItemCount() == 0));
+        mDownloadButton.setEnabled(!(ListViewUtils.selectedItemCount(getListView()) == 0));
```

Comment:
I intentionally made as few changes to the previous code as possible. In this case, specifically just substituted the refactored (moved-to-ListViewUtils) method and kept the check the same as before.

Your suggestion is a good one, but probably should be carried out throughout the code base in a different commit, no?